include/mysql/plugin_audit.h

```diff
@@ -102,6 +102,8 @@ struct mysql_event_connection
   const char *ip;
   unsigned int ip_length;
   MYSQL_CONST_LEX_STRING database;
+  const char *tls_version;
+  unsigned int tls_version_length;
```

Comment:
this changes the audit plugin API (as highlighted by the fact that you had to change the `.pp` file) — meaning, you need to change the `MYSQL_AUDIT_INTERFACE_VERSION`.

it's a backward compatible change, so please increment the second version component: 0x0302 → 0x0303.

Make sure to run all tests after that, there might be a couple of tests that show this version — they'll need to be updated. Or grep for the version in all result files at least.

Reply:
Done and verified